front/src/applications/operationalStudies/components/MacroEditor/nodeStore.ts

```diff
@@ -14,7 +14,10 @@ const loadNodes = () => {
   let entries = [];
   const rawNodes = localStorage.getItem(NODES_LOCAL_STORAGE_KEY);
   if (rawNodes) {
-    entries = JSON.parse(rawNodes);
+    const parsedNodes = JSON.parse(rawNodes);
+    if (Array.isArray(parsedNodes)) {
+      entries = parsedNodes;
+    }
```

Comment:
Can we log an error if it ends up not being an array? Just to let the dev know that we didn't actually load anything from the localStorage.